pkg/apis/serving/v1alpha2/kfservice_types.go

```diff
@@ -88,6 +88,7 @@ type ExplainerSpec struct {
 // TransformerSpec defines transformer service for pre/post processing
 type TransformerSpec struct {
 	Custom *CustomSpec `json:"custom,omitempty"`
+	RequestLogging *v1.ObjectReference `json:"requestLogging,omitempty"`
```

Comment:
It can an annotation on the InferenceService spec imo, for now we put in Transformer we may change the place where do request logging later on. Maybe we just need a flag, since we can actually automatically provision the channel under the hood with a well-known channel name or send the request to the default eventing broker in the user namespace.

Reply:
Good idea. I'll leave PR here to discuss Weds and can replace with impl PR.